src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/ReportedAdjudicationServiceTest.kt

```diff
@@ -557,6 +784,66 @@ class ReportedAdjudicationServiceTest {
         verify(prisonApiGateway).publishAdjudication(expectedAdjudicationToPublish)
       }
     }
+
+    @Nested
```

Comment:
We can remove this and other nested class here if we think the parameterised tests are better